Sources/Parma/Element/ListAttributes.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ListAttributes.swift
+//  Parma
+//
+//  Created by Antoine Lamy on 5/13/21.
+//
+//  Copyright (c) 2020 Leonard Chan <wxclx98@gmail.com>
+//
+//  MIT license, see LICENSE file for details
+
+import Foundation
+
+/// Attributes used to display a list element.
+public struct ListAttributes {
+    var type: ListType
+    var delimiter: ListDelimiter
```

Comment:
Please make these two attributes public, otherwise, they cannot be assessed outside. And can you add some comments to those? Thanks.

Reply:
Done ✅